docs/changelog/82819.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 82819
+summary: "[Draft] Adjust `indices.recovery.max_bytes_per_sec` according to external\
```

Comment:
Can we remove `[Draft]` here (and preferably in 8.1 too)?

Reply:
I opened #83527 for 8.1, thanks for catching this